01week/bio/index.html

```diff
@@ -19,12 +19,11 @@
     <br>
     <p><b>I'm a student at Austin Coding Academy and a fan of chickens... the pet that makes you breakfast.</b></p>
```

Comment:
manny- thanks so much for your review, positive feedback, and suggestion! i will check out the links, i need allll the information i can get. thanks again! : )